react-app/src/pages/QuizComponent/QuizCard/QuizCard.css

```diff
@@ -0,0 +1,29 @@
+.quiz-card {
```

Comment:
With CSS Modules, please use camelCase for the style names! Same with lines 17, 22, and 27